test/kubernetes/e2e/features/deployer/suite.go

```diff
@@ -181,6 +185,11 @@ func (s *testingSuite) TestProvisionResourcesUpdatedWithValidParameters() {
 	// the GatewayParameters modification should cause the deployer to re-run and update the
 	// deployment to have 2 replicas
 	s.testInstallation.Assertions.EventuallyRunningReplicas(s.ctx, proxyDeployment.ObjectMeta, gomega.Equal(2))
+
+	// modify the external traffic policy in the GatewayParameters
+	s.patchGatewayParameters(gwParamsDefault.ObjectMeta, func(parameters *v1alpha1.GatewayParameters) {
```

Comment:
seems like there should be a check after this to ensure that eventually the provisioned service contains the new ext traffic policy